plenum/server/request_handlers/handler_interfaces/request_handler.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABCMeta, abstractmethod
+from plenum.common.request import Request
+from stp_core.common.log import getlogger
+
+logger = getlogger()
+
+
+class RequestHandler(metaclass=ABCMeta):
+
+    @abstractmethod
+    def static_validation(self, request: Request):
+        """
+        Does static validation like presence of required fields,
+        properly formed request, etc
+        """
+
+    @abstractmethod
+    def dynamic_validation(self, request: Request):
```

Comment:
It makes sense only for the requests being ordered, that is write requests

Reply:
removed